### ISSUE TITLE
Intermittent failure in InstructorFeedbackEditPageUiTest

### ISSUE THREAD

**Issue by tarun14110** [TEAMMATES/teammates#4949]
browsertests.InstructorFeedbackEditPageUiTest
```
org.openqa.selenium.TimeoutException: Timed out after 800 seconds waiting for presence of element located by: By.className: ui-datepicker-month
Build info: version: '2.46.0', revision: '87c69e2', time: '2015-06-04 16:16:47'
System info: host: 'mafia-Inspiron-3542', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'amd64', os.version: '3.16.0-57-generic', java.version: '1.7.0_80'
Session ID: 522bf2d0-fe5e-4480-b074-d2e6887696a0
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=LINUX, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, browserConnectionEnabled=true, nativeEvents=false, webStorageEnabled=true, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=28.0}]
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:80)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:229)
	at teammates.test.pageobjects.AppPage.waitForElementPresence(AppPage.java:222)
	at teammates.test.pageobjects.InstructorFeedbackEditPage.getDatepickerMonth(InstructorFeedbackEditPage.java:640)
	at teammates.test.pageobjects.InstructorFeedbackEditPage.navigate(InstructorFeedbackEditPage.java:618)
	at teammates.test.pageobjects.InstructorFeedbackEditPage.areDatesOfPreviousCurrentAndNextMonthEnabled(InstructorFeedbackEditPage.java:556)
	at teammates.test.pageobjects.InstructorFeedbackEditPage.areDatesOfPreviousCurrentAndNextMonthEnabled(InstructorFeedbackEditPage.java:540)
	at teammates.test.cases.ui.browsertests.InstructorFeedbackEditPageUiTest.testEditSessionAction(InstructorFeedbackEditPageUiTest.java:139)
	at teammates.test.cases.ui.browsertests.InstructorFeedbackEditPageUiTest.allTests(InstructorFeedbackEditPageUiTest.java:81)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"class name","selector":"ui-datepicker-month"}
Command duration or timeout: 5 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.46.0', revision: '87c69e2', time: '2015-06-04 16:16:47'
System info: host: 'mafia-Inspiron-3542', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'amd64', os.version: '3.16.0-57-generic', java.version: '1.7.0_80'
*** Element info: {Using=class name, value=ui-datepicker-month}
Session ID: 522bf2d0-fe5e-4480-b074-d2e6887696a0
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=LINUX, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, browserConnectionEnabled=true, nativeEvents=false, webStorageEnabled=true, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=28.0}]
	at sun.reflect.GeneratedConstructorAccessor58.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:156)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:605)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:358)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByClassName(RemoteWebDriver.java:439)
	at org.openqa.selenium.By$ByClassName.findElement(By.java:388)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:350)
	at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:809)
	at org.openqa.selenium.support.ui.ExpectedConditions.access$0(ExpectedConditions.java:807)
	at org.openqa.selenium.support.ui.ExpectedConditions$6.apply(ExpectedConditions.java:181)
	at org.openqa.selenium.support.ui.ExpectedConditions$6.apply(ExpectedConditions.java:1)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:209)
	... 20 more
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"class name","selector":"ui-datepicker-month"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.46.0', revision: '87c69e2', time: '2015-06-04 16:16:47'
System info: host: 'mafia-Inspiron-3542', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'amd64', os.version: '3.16.0-57-generic', java.version: '1.7.0_80'
Driver info: driver.version: unknown
	at <anonymous class>.FirefoxDriver.prototype.findElementInternal_(file:///tmp/anonymous763494744482487442webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:10299)
	at <anonymous class>.FirefoxDriver.prototype.findElement(file:///tmp/anonymous763494744482487442webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:10308)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_/h(file:///tmp/anonymous763494744482487442webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12282)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_(file:///tmp/anonymous763494744482487442webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12287)
	at <anonymous class>.DelayedCommand.prototype.execute/<(file:///tmp/anonymous763494744482487442webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12229)
```

**Comment by wkurniawan07** [comment]
Let's take it one failure per issue. Besides, the 2nd one is already addressed in another issue.

**Comment by wkurniawan07** [comment]
Anyway, this failure has a quick fix: ensure that the browser running this test is on the foreground when you're running it. It will pass. Also, 800 seconds timeout is waaaaaay too long. 25-30 seconds is typically enough for dev server.

**Comment by tarun14110** [comment]
@wkurniawan07 it worked for me.

**Comment by wkurniawan07** [comment]
I can't promise anything because it seems like everything has been tried to fix this issue, but I'll take a look.

**Comment by kanghj** [comment]
Actually, this was probably fixed before but accidentally removed in #4663.
